### ISSUE TITLE
Once event is rejected unblock corresponding rooms

### ISSUE THREAD

**Issue by rsarky** [cribbl/miteb-frontend#66]
**Is your feature request related to a problem? Please describe.**
Once rejected by an authority the corresponding rooms should be unblocked.

**Comment by rsarky** [comment]
@nids98 This will be an easy one. Once the event is rejected by an authority delete the corresponding rooms from the `roomsx` node so that they are freed

**Comment by rsarky** [comment]
Hey @nids98 . Are you still working on this?

**Comment by nids98** [comment]
I did. But I messed up the branch, will create a new pr.

**Comment by rsarky** [comment]
Sure.Try to do this asap because it's a pretty important feature

**Comment by rsarky** [comment]
Ping @nids98

**Comment by rsarky** [comment]
I can probably pick this up and merge as this is pretty important
